docs/source/graphs.rst

```diff
@@ -42,6 +42,9 @@ execute this graph in a way that respects these data dependencies and leverages
 parallelism where possible, multiple independent tasks can be run
 simultaneously.
 
+.. image:: images/map-reduce-task-scheduling.svg
+   :align: center
```

Comment:
In the future, we'll be flagging these for alt text, for now we can deliberately defer :)

Reply:
Since Dask is represented by the "Full Task Scheduling" box in this image, perhaps we can name check that further down?

Perhaps something like

```
Dask is a specification that encodes full task scheduling with minimal incidental...
```
(fixes a typo as a bonus)

Reply:
done!